specs/core/0_beacon-chain.md

```diff
@@ -1526,10 +1551,33 @@ For each `attestation` in `block.body.attestations`:
 * Verify that `attestation.data.justified_slot` is equal to `state.justified_slot if attestation.data.slot >= state.slot - (state.slot % EPOCH_LENGTH) else state.previous_justified_slot`.
 * Verify that `attestation.data.justified_block_root` is equal to `get_block_root(state, attestation.data.justified_slot)`.
 * Verify that either `attestation.data.latest_crosslink_root` or `attestation.data.shard_block_root` equals `state.latest_crosslinks[shard].shard_block_root`.
-* `aggregate_signature` verification:
-    * Let `participants = get_attestation_participants(state, attestation.data, attestation.aggregation_bitfield)`.
-    * Let `group_public_key = bls_aggregate_pubkeys([state.validator_registry[v].pubkey for v in participants])`.
-    * Verify that `bls_verify(pubkey=group_public_key, message=hash_tree_root(AttestationDataAndCustodyBit(attestation.data, False)), signature=attestation.aggregate_signature, domain=get_domain(state.fork, attestation.data.slot, DOMAIN_ATTESTATION))`.
+* Verify bitfields and aggregate signature:
+
+```python
+    assert attestation.custody_bitfield == 0  # [TO BE REMOVED IN PHASE 1]
```

Comment:
Doesn't this line still need to be removed?

Reply:
Changing it to `assert attestation.custody_bitfield == b'\x00' * len(attestation.custody_bitfield)`. In phase 0, don't we want to enforce all custody bits to be zero?